doc/source/index.rst

```diff
@@ -28,11 +35,10 @@ Ansys Electronics Desktop (AEDT) to make scripting simpler.
    .. grid-item-card::
             :img-top: _static/assets/index_user_guide.png
 
-            User guide
-            ^^^^^^^^^^
+            Tutorials
```

Comment:
Again, I strong advise that you keep this "User guide" to keep it consistent with the documenation for other PyAnsys libraries.